bee-api/bee-rest-api/src/types/dtos.rs

```diff
@@ -863,9 +866,9 @@ impl TryFrom<&MigratedFundsEntry> for MigratedFundsEntryDto {
 
     fn try_from(value: &MigratedFundsEntry) -> Result<Self, Self::Error> {
         Ok(MigratedFundsEntryDto {
-            tail_transaction_hash: value.tail_transaction_hash().as_ref().into(),
+            tail_transaction_hash: hex::encode(value.tail_transaction_hash().as_ref()),
```

Comment:
Double-checking: is this supposed to be hex ? Or a tryte string as this is from the legacy network ?

Reply:
Not sure, I just know that we get `e0ce1933a900d02e0a43e2ba4e922d1a706a36a117ce1eb9ad0a74d39ee722c41b00cd227232efd76f5997e91bf8000000` from HORNET